From 4a43a85eadd08231a4dc5f79720e5fa23b555d31 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Mon, 7 Mar 2016 14:53:21 +0100
Subject: [PATCH] score: Add panic timeout to kernel parameters

Add panic timeout 2 to kernel parameters. This can be changed by the
running system.

Ticket #1008
---
 include/configs/mx6sxscore.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/mx6sxscore.h b/include/configs/mx6sxscore.h
index 96dc728..1871918 100644
--- a/include/configs/mx6sxscore.h
+++ b/include/configs/mx6sxscore.h
@@ -178,7 +178,7 @@
     "nandkernel=nand read ${loadaddr} 0x80000 0x800000\0" \
     "nandfdt=nand read ${fdt_addr} 0x880000 0x100000\0" \
     "nandargs=setenv bootargs " \
-        "console=${console},${baudrate} uart_from_osc ubi.mtd=5 root=${rootnand} mtdparts=${mtdparts}\0" \
+        "console=${console},${baudrate} panic=2 uart_from_osc ubi.mtd=5 root=${rootnand} mtdparts=${mtdparts}\0" \
     "bootnand=run getrootname && " \
         "run setrootnand && " \
         "run nandargs && " \
-- 
1.9.1

